apps/dcos_l4lb/src/dcos_l4lb_mesos_poller.erl

```diff
@@ -88,24 +89,40 @@ handle_poll() ->
 handle_poll(false) ->
     ok;
 handle_poll(true) ->
+    Begin = erlang:monotonic_time(),
     try dcos_net_mesos_listener:poll() of
         {error, Error} ->
+            prometheus_counter:inc(
+                l4lb, poll_failures_total,
```

Comment:
`dcos_net_mesos_listener:poll/0` returns error only if `dcos_net_mesos:call/1` fails. It means `call_failures_total` metric already covers this case.

Reply:
So do you propose we remove `poll_failures_total`, or is that another place you would recommend putting it?

Reply:
I'd remove it.